api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1235,6 +1237,10 @@ public static String getOpenmrsProperty(String property) {
 	
 	public static final String LOG_LEVEL_FATAL = "fatal";
 	
+	public static final String LOGIN_URL = "login.htm";
+	
+	public static final String LOGIN_URL_PROPERTY = "LOGIN_URL";
```

Comment:
Did you take a look at this class to see the convention used for naming other global property constants?

Reply:
had not, now i have